app/models/user.rb

```diff
@@ -53,7 +53,8 @@ class User < ActiveRecord::Base
 
   scope :admin, -> { where(is_admin: true) }
 
-  validates :email, :password, :password_confirmation, presence: true
+  validates :email, presence: true
+  validates :password, :password_confirmation, presence: true, on: :create
```

Comment:
this doesn't work neither. If you want to change your password you are not creating the user, but updating it and it should also be validated. Have you read https://github.com/openSUSE/osem/pull/1408#discussion_r108110859?

As you can read in the [Rails guide](http://guides.rubyonrails.org/active_record_validations.html#confirmation), this should be done like:

``` Ruby
validates :email, presence: true

validates :password, confirmation: true
validates :password_confirmation, presence: true
```